src/c/transport/transport.c

```diff
@@ -25,21 +25,17 @@ void transport_init(struct rasta_handle *h, rasta_transport_channel *channel, un
 
 // finds the transport channel corresponding to the sender (identified by IP address and port)
 // and returns the indexes of this transport channel and the redundancy channel it belongs to
-void find_channel_by_ip_address(struct rasta_handle *h, struct sockaddr_in sender, int *red_channel_idx, int *transport_channel_idx) {
-    *red_channel_idx = -1;
+void find_channel_by_ip_address(struct rasta_handle *h, struct sockaddr_in sender, int *transport_channel_idx) {
```

Comment:
Now that we only have one return value here, we could return it directly instead of doing pointer stuff (even better if we could directly return the `rasta_transport_channel`, but I had some problems implementing that last year).